Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs

```diff
@@ -336,6 +372,14 @@ void UpdateContent()
 			AddView(_contentView, new LayoutParams(LayoutParams.MatchParent, LayoutParams.MatchParent));
 		}
 
+		void UpdateSwipeItems()
+		{
+			_leftItems = Element.LeftItems;
```

Comment:
Can this be null?

Reply:
No, by default it is an empty collection.